modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assembly:
+//
+// * configuring-an-openshift-cluster-by-deploying-an-application-with-cluster-configurations.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-additional-permissions-for-cluster-config_{context}"]
+= Providing additional permissions for cluster configuration
+
+For better cluster-configuration management you can add additional permissions to an Argo CD instance. 
+
+.Procedure
+
+For additional permissions, create **Cluster Roles** with additional permissions and then a new **Cluster Role Binding** to associate them to a **ServiceAccount**:
```

Comment:
You write about ClusterRoles and ClusterRoleBindings here, but in the examples below, you use Roles and RoleBindings.